src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -3845,6 +3851,11 @@ inline void rpp_store12_i8pkd3_to_i8pln3(Rpp8s* dstPtrR, Rpp8s* dstPtrG, Rpp8s*
 }
 
 inline void rpp_store4_i8_to_i8(Rpp8s* dstPtr, __m128i &p)
+{
```

Comment:
I suggest not to do these kinds of calling convensions which calls single line inline functions from another functions. This will make calling inefficient. Please avoid